contracts/TomoTokenSale.sol

```diff
@@ -405,32 +369,36 @@ contract TomoTokenSale is Pausable {
 
     // running while PoC Buying Time
     if (now <= fundingPoCEndTime) {
-      tokensToAllocate = cap.sub(token.balanceOf(msg.sender));
+      tokensToAllocate = cap.sub(participated[_beneficiary]);
     } else {
-      tokensToAllocate = maxCap.sub(token.balanceOf(msg.sender));
+      tokensToAllocate = maxCap.sub(participated[_beneficiary]);
     }
 
     // calculate refund if over max cap or individual cap
     if (tokens > tokensToAllocate) {
       tokensToRefund = tokens.sub(tokensToAllocate);
```

Comment:
(optional) I find `token` and `tokens` confusing because they refer to different object types

Reply:
ok